server/src/ability/ability.util.ts

```diff
@@ -61,6 +61,10 @@ const simpleAbilityCheck: OperationAbilityChecker = async (
 ) => {
   // Extract entity name from model name
   const entity = camelCase(modelName);
+  //TODO: Fix boolean data types so that disconnects are possible
+  if (typeof data === 'boolean') {
```

Comment:
Yes, you are right, this is correct!

Reply:
I'm not too well versed in prisma so I am unsure of a solution, if you have some guidance on solving it I would gladly include it in my next commit/push